packages/components/src/pricing-slider/index.tsx

```diff
@@ -21,6 +20,7 @@ const PricingSlider: React.FC< PricingSliderProps > = ( {
 	onBeforeChange,
 	onAfterChange,
 	renderThumb,
+	marks, // Add marks to the component props
```

Comment:
Should we default it to `false` so that it doesn't affect the other place that use the component?

Reply:
Yes! The underlying slider handles "undefined" correctly (ie: not showing anything) but I don't think it hurts for us to be explicit in our wrapper.